plugin/trino-tpch/src/main/java/io/trino/plugin/tpch/TpchMetadata.java

```diff
@@ -575,6 +575,15 @@ private static String scaleFactorSchemaName(double scaleFactor)
         return "sf" + scaleFactor;
     }
 
+    private static String scaleFactorToNormalizedSchemaName(double scaleFactor)
+    {
+        if (scaleFactor == TINY_SCALE_FACTOR) {
+            return TINY_SCHEMA_NAME;
+        }
+
+        return "sf" + (int) scaleFactor;
```

Comment:
Would lose information for non-integer scale factors.

Reply:
I see, good catch. I'll add some more cases to the tests.

Reply:
I have updated the code and added new tests, info so no longer be lost.